### PR TITLE
feat: mocking date

### PR DESCRIPTION
**Pull request by sheremet-va**
#249

### CODE REVIEW

File: docs/guide/mocking-date.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+All useful methods are located on `vi` object that you can import from `vitest` package or access globally, if you have [`global`](/config/#global) config enabled.
+
+## setSystemDate
+
+- **Type**: `(date: string | number | Date) => void`
+
+Sets current date to the one that was passed. All `Date` calls will return this date.
+
+Useful if you need to test anything that depends on current date - for example `momentjs` calls inside your code.
```
Comment (patak-dev) [inline docs/guide/mocking-date.md:13]: Let's push people to more modern alternatives
```suggestion
Useful if you need to test anything that depends on the current date - for example [luxon](https://github.com/moment/luxon/) calls inside your code.
```

**Review by patak-dev — APPROVED** [review]

**Comment by antfu** [comment]
Could we have some tests for it?

**Comment by sheremet-va** [comment]
> Could we have some tests for it?

Added some tests.

**Comment by sheremet-va** [comment]
> > Could we have some tests for it?
> 
> Added some tests. I think they are enough

And they are failing in CI! What a delight

**Comment by sheremet-va** [comment]
> > > Could we have some tests for it?
> > 
> > 
> > Added some tests. I think they are enough
> 
> And they are failing in CI! What a delight

Fixed now :p

**Pull request merged**